common/src/View/LayerEditor.cpp

```diff
@@ -329,12 +330,60 @@ namespace TrenchBroom {
             document->resetVisibility(std::vector<Model::Node*>(std::begin(layers), std::end(layers)));
         }
 
+        bool LayerEditor::canShowAllLayers() const {
+            for (const auto* layer : kdl::mem_lock(m_document)->world()->allLayers()) {
```

Comment:
I'd also use std::any_of here and in the other conditions. I've seen you use in your other PRs. Don't try to deny it! :-P